packages/front-generator/src/generators/react-typescript/app/template/src/app/common/Spinner.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Spin} from "antd";
+
+export function Spinner() {
+
+  return (
+    <div
+      style={{
+        position: "absolute",
+        left: "50%",
+        top: "50%",
+        transform: "translate(-50%, -50%)"
+      }}
```

Comment:
I think it's best if we avoid inline styles completely and only use RSCSS-conforming classes. This specific set of styles might be a candidate for a [Helper](https://rscss.io/helpers.html).

Reply:
Seems to be my fault actually, I seem to have missed a few occurrences of inline styles when I was implementing the methodology. I'll fix the remaining occurrences in #78, would you please be so kind to fix the spinner as part of this ticket? Thanks.

Reply:
We decide to refactor css in spinner in #78.